visualization/dashboard/src/MLIDashboard/Cohort.ts

```diff
@@ -113,13 +113,20 @@ export class Cohort {
                 const rowVal = row[filter.column];
                 switch(filter.method){
                     case FilterMethods.equal:
-                        return rowVal === filter.arg;
+                        return rowVal === filter.arg[0];
                     case FilterMethods.greaterThan:
-                        return rowVal > filter.arg;
+                        return rowVal > filter.arg[0];
+                    case FilterMethods.greaterThanEqualTo:
+                        return rowVal >= filter.arg[0];
                     case FilterMethods.lessThan:
-                        return rowVal < filter.arg;
+                        return rowVal < filter.arg[0];
+                    case FilterMethods.lessThanEqualTo:
+                        return rowVal <= filter.arg[0];
                     case FilterMethods.includes:
                         return (filter.arg as number[]).includes(rowVal);
+                    case FilterMethods.inTheRangeOf:
+                        //TODO:validate arg[0] < arg[1]
```

Comment:
validation can be in the view, since the filter still makes sense if not true, just no value will satisfy the filter (which makes sense). Won't break or anything,

Reply:
added validation in View (setNumericValue which is responsible for setting values in the SpinBox)